src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -11,12 +12,19 @@ internal class EndToEndTraceHelper
         private static string appName;
         private static string slotName;
 
-        // TODO: Replace TraceWriter with structured event client
         private readonly TraceWriter appTraceWriter;
+        private readonly ILogger logger;
+        private readonly bool isAppInsightsLoggingEnabled;
```

Comment:
I don't think you need to track this at all -- you can already fully disable tracing externally. I worry that having two ways to control logging can lead to some confusion.

e.g. you can do this to fully turn off logs from this trigger (assuming this is the category name):
```
{
  "logger": {
    "categoryFilter": {
      "categoryLevels": {
        "Host.Triggers.DurableTask": "None"
      }
    }
}
```